src/python/pants/option/options_bootstrapper.py

```diff
@@ -262,7 +265,7 @@ def verify_configs_against_options(self, options: Options) -> None:
                     all_options_under_scope = set(config.values.options(section)) - set(
                         config.values.defaults
                     )
-                    for option in all_options_under_scope:
+                    for option in sorted(all_options_under_scope):
```

Comment:
Not relevant to the flake, but this means our config validation error will now be deterministic!

Reply:
yay determinism!